pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -376,6 +378,17 @@ func (r *rbac) handleSCCInNamespace(ctx context.Context, ns *corev1.Namespace) e
 	return nil
 }
 
+func getRBACMaxCalls() int {
```

Comment:
Changing an environment variable will lead to the recreation of the pod. So this can be part of init where we change defaultRbacMaxConcurrentCalls if environment variable is provided.